docs/databases/postgres.md

```diff
@@ -4,4 +4,118 @@ title: Using PostgreSQL with Graphback
 sidebar_label: PostgreSQL
 ---
 
-## TODO
\ No newline at end of file
+[PostgreSQL](https://www.postgresql.org/) is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance. [Graphback Knex Provider](https://www.npmjs.com/package/@graphback/runtime-knex) package provides instant integration to your PostgreSQL database, giving you a full implementation of the [CRUD API](../crud/introduction.md). 
+
+Additionally, Graphback comes with the [GraphQL Migrations](../graphql-migrations/intro.md) package that can be used to perform migrations against your database using the business models defined in the GraphQL Schema. 
+
+The provider is built on top of [Knex.js](http://knexjs.org/), a flexible SQL query builder.
+
+# Installation
+
+Install with npm:
+
+```bash
+npm install @graphback/runtime-knex knex
+```
+
+Install with yarn:
+
+```bash
+yarn add @graphback/runtime-knex knex
+```
+
+Running either of the two commands will install the `Graphback Knex Provider` ([`@graphback/runtime-knex`](https://www.npmjs.com/package/@graphback/runtime-knex)) and the Knex.js library ([`knex`](https://www.npmjs.com/package/knex)).
```

Comment:
Link to Knex.js website instead?

Reply:
We have the link here https://github.com/aerogear/graphback/pull/1698/files#diff-dd237a6c711114adc64df9e89cb0f528R11 since here we are talking about the package thought we could have the npm link insted